pkg/collector/corechecks/snmp/internal/checkconfig/config_test.go

```diff
@@ -246,9 +245,7 @@ bulk_max_repetitions: 20
 		{Tag: "snmp_host", OID: "1.3.6.1.2.1.1.5.0", Name: "sysName"},
 	}
 
-	expectedMetricsJSON, _ := json.MarshalIndent(expectedMetrics, "", "\t")
-	actualMetricsJSON, _ := json.MarshalIndent(config.Metrics, "", "\t")
-	assert.JSONEq(t, string(expectedMetricsJSON), string(actualMetricsJSON))
+	assert.Equal(t, expectedMetrics, config.Metrics)
```

Comment:
Note for reviewers: changing to `assert.Equal` since we use lists in json and the order of the list is not deterministic

Reply:
> Is this actually deterministic? We generate it by doing for key, value := range lm and I thought go maps iterated in a random order.

@dplepage-dd Do you mean deterministic for the metric_tags mappings (ListMap)?
If so, we are comparing here `map[string]string`, so the order shouldn't matter, isn't it?